src/main/java/com/netflix/imflibrary/st0377/HeaderPartition.java

```diff
@@ -194,9 +194,13 @@ public HeaderPartition(ByteProvider byteProvider, long byteOffset, long maxParti
                             prefaceSetCount));
         }
 
-        if (imfErrorLogger.getNumberOfErrors() > numErrors)
+        if (imfErrorLogger.getNumberOfErrors() > 0)
         {
-            throw new MXFException(String.format("%d errors encountered when reading header partition", imfErrorLogger.getNumberOfErrors() - numErrors));
```

Comment:
Makes sense, since at this point we might flag an exception only if there are any errors accumulated while parsing and reading the header partition.